compiler/rustc_trait_selection/src/traits/misc.rs

```diff
@@ -42,6 +44,10 @@ pub fn can_type_implement_copy<'tcx>(
             _ => return Err(CopyImplementationError::NotAnAdt),
         };
 
+        if let hir::ImplPolarity::Negative(_) = polarity {
+            return Ok(());
+        }
```

Comment:
it seems acceptable to only limit negative `Copy` impls on coherence, is it?

while pointless, I don't see why we should forbid
```rust
extern {
    type Extern;
}

impl !Copy for Extern {}
```

Reply:
All right, makes sense. That's what I did initially but I wasn't really sure. So I changed it. Changing it back…